packages/kotti-banner/src/Banner.vue

```diff
@@ -65,7 +65,7 @@ export default {
 			}
 		},
 		expandable() {
-			return this.$slots.expand
+			return !!this.$slots.expand
```

Comment:
henlo pls Boolean(thing) ty

Reply:
Why is this code also in #125?

Reply:
#124 u mean. idk what is #125 icet
and yes i just told u i reset head and made a separate issue (126)

will commit this here now.